example/gluon/word_language_model/train.py

```diff
@@ -22,11 +22,13 @@
 from mxnet import gluon, autograd
 from mxnet.gluon import contrib
 from mxnet.gluon.model_zoo.text.lm import RNNModel
-import data
+from mxnet.gluon.model_zoo.text.lm import AWDLSTM
 
 parser = argparse.ArgumentParser(description='MXNet Autograd RNN/LSTM Language Model on Wikitext-2.')
 parser.add_argument('--model', type=str, default='lstm',
                     help='type of recurrent net (rnn_tanh, rnn_relu, lstm, gru)')
+parser.add_argument('--awd', type=str, default='true',
+                    help='Whether to apply awd (true, false)')
```

Comment:
remove `--awd` flag, and add option for AWD model in the `--model` string instead.